packages/dnb-design-system-portal/src/docs/uilib/components/date-picker/Examples.tsx

```diff
@@ -51,8 +51,9 @@ export const DatePickerRange = () => (
       on_cancel={({ start_date, end_date }) => {
         console.log('on_cancel', start_date, end_date)
       }}
-      onBlur={({ start_date, end_date }) => {
-        console.log('onBlur', start_date, end_date)
+      onBlur={({ start_date, end_date, partial_start_date, partial_end_date }) => {
+        console.log('onBlurPartial', partial_start_date, partial_end_date)
```

Comment:
```suggestion
        console.log('onBlurPartial', partialStartDate, partialEndDate)
```
Not sure which order the comments are being displayed in, but the same reasoning as given below, or above when it comes to future proofing with camelCasing 😄